### PR TITLE
Pioneer 20e gliders

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: deployment/CP15MOAS-GL380_Deploy.csv
Comment (reedan88) [inline deployment/CP15MOAS-GL380_Deploy.csv:1]: Looks good

File: deployment/CP15MOAS-GL559_Deploy.csv
Comment (reedan88) [inline deployment/CP15MOAS-GL559_Deploy.csv:1]: Looks good.

File: deployment/CP15MOAS-GL379_Deploy.csv
Comment (reedan88) [inline deployment/CP15MOAS-GL379_Deploy.csv:1]: Looks good.

File: deployment/CP15MOAS-GL388_Deploy.csv
Comment (reedan88) [inline deployment/CP15MOAS-GL388_Deploy.csv:1]: Looks good.

File: deployment/CP15MOAS-GL335_Deploy.csv
Comment (reedan88) [inline deployment/CP15MOAS-GL335_Deploy.csv:1]: Looks good.

File: deployment/CP15MOAS-PG514_Deploy.csv
Comment (reedan88) [inline deployment/CP15MOAS-PG514_Deploy.csv:1]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good. No error spotted.

File: deployment/CP15MOAS-PG514_Deploy.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
+PH20241119,,,,CP15MOAS-PG514-00-ENG000000,1,1,2024-11-20T16:55:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00514,CGCON-GLDRCP-00514,35.95962,-74.75837,,1000,1000,OF,,
+PH20241119,,,,CP15MOAS-PG514-01-CTDGVM000,1,1,2024-11-20T16:55:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00514,CGINS-CTDGVM-09301,35.95962,-74.75837,,1000,1000,OF,,
+PH20241119,,,,CP15MOAS-PG514-02-DOSTAM000,1,1,2024-11-20T16:55:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00514,CGINS-DOSTAM-00543,35.95962,-74.75837,,1000,1000,OF; top mount,,
+PH20241119,,,,CP15MOAS-PG514-03-FLORTM000,1,1,2024-11-20T16:55:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00514,CGINS-FLORTM-04101,35.95962,-74.75837,,1000,1000,OF,,
+PH20241119,,,,CP15MOAS-PG514-04-FLORTO000,1,1,2024-11-20T16:55:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00514,CGINS-FLORTO-01342,35.95962,-74.75837,,1000,1000,OF,,
+PH20241119,,,,CP15MOAS-PG514-05-NUTNRM000,1,1,2024-11-20T16:55:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00514,CGINS-NUTNRM-00661,35.95962,-74.75837,,1000,1000,OF,,
+PH20241119,,,,CP15MOAS-PG514-06-PARADM000,1,1,2024-11-20T16:55:00,,CGVCP-15MOAS-00000,CGVEH-GLDRCP-00514,CGINS-PARADM-50217,35.95962,-74.75837,,1000,1000,OF,,
```
Comment (cooleyky) [inline deployment/CP15MOAS-PG514_Deploy.csv:8]: looks good

**Review by cooleyky — APPROVED** [review]
looks good

**Pull request merged**